client-src/elements/external-reviewers.ts

```diff
@@ -108,9 +107,7 @@ export class ExternalReviewer {
    * @param {Record<string, LabelInfo>} labels
    */
   constructor(icon, labels) {
```

Comment:
Also no strong need to do this in these initial PRs, but once you're migrating in bulk, you should replace `@param` comments with Typescript types on the parameters, like:

```suggestion
   */
  constructor(icon, labels: Record<string, LabelInfo>) {
```